cluster/manifests/skipper/deployment.yaml

```diff
@@ -447,10 +447,6 @@ spec:
           - "-kubernetes-east-west-range-domains=ingress.cluster.local"
           - "-kubernetes-east-west-range-predicates=ClientIP(\"10.2.0.0/15\", \"{{ .Values.vpc_ipv4_cidr }}\")"
           - "-reverse-source-predicate"
-          - "-enable-api-usage-monitoring"
-          - "-api-usage-monitoring-realm-keys=https://identity.zalando.com/realm"
-          - "-api-usage-monitoring-client-keys=https://identity.zalando.com/managed-id,sub"
-          - "-api-usage-monitoring-default-client-tracking-pattern=services[.].*"
```

Comment:
Why do we remove this?

Reply:
I don't see it used in routesrv

Reply:
I guess we need to enable it in skipper unconditionally then

Reply:
`-default-filters-dir` is used in routesrv options but api usage isn't

Reply:
but why? DefaultFilters will fill k8s data client and skipper will validate

Reply:
Default filters should stay in routesrv and @MustafaSaber do not drop the options as it feeds into the same tool apimonitoring, which configures the apiMonitoring filters. We want to drop these from skipper-ingress and put it into routesrv only.

Reply:
Ok, apiUsageMonitoring and defaultFIltersDir are related but I think it will work like you propose

Reply:
@szuecs options for apiMonitoring weren't in place the flag is provided in config but isn't collected from routesrv

Reply:
I would guess that we need this in both to make it work. The default-filters-dir is likely the one we can drop from skipper-ingress.
Routesrv likely requires the configuration, too, because it parses the kubernetes resources and creates the routes and then outputs eskip.
Skipper-ingress should only pick up the eskip with the same configuration to create the correct filter again.